src/seasidecache.cpp

```diff
@@ -1420,6 +1438,7 @@ void SeasideCache::startRequest(bool *idleProcessing)
             m_fetchRequest.setFilter(favoriteFilter());
             m_fetchRequest.setFetchHint(favoriteFetchHint(m_fetchTypes));
             m_fetchRequest.setSorting(m_sortOrder);
+            qDebug() << "Starting favorites query at" << m_timer.elapsed() << "ms";
```

Comment:
yeah but unfortunately most SELECT statements are reported as -1 because the statement isn't finished yet when the call returns. (the rows still have to be read and the caller is in control if that)